### ISSUE TITLE
Imperium - Space Wolves: WGPL in Long Fangs should be able to swap storm bolter when equipped with Cyclone

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#7150]
**TL;DR:** A WGPL in Terminator Armour in a Long Fang squad should be able to swap his storm bolter after taking a Cyclone ML.

**Description:** Wolf Guard in Terminator Armour with Heavy weapon:
when 'cyclone missile launcher and storm bolter' are selected as one option, this option doesn't allow to replace the storm bolter afterwards.
as per the Space Wolves faq THIS SHOULD BE POSSIBLE:
FAQ SPACE WOLVES:
Q: If I replace a Wolf Guard Terminator’s storm bolter with a
cyclone missile launcher and storm bolter from the Terminator
Heavy Weapons list (as per the third bullet point), can I
subsequently replace the new storm bolter with an item from the
Combi-weapons or Terminator Melee Weapons list (as per
the second bullet point)?
A: Yes.

so, this option should allow any Wolf Guard in Terminator Armour (including the Terminator Wolf Guards in LONGFANGS units) to be able to equip a storm shield plus a thunder hammer (or any other weapon melee or not) even if they're wearing a Cyclone missile launcher. Now it's impossible to avoid using a storm bolter.
